--- source-html.vim	2017-07-05 21:26:08.000000000 +0800
+++ html.vim	2017-07-05 22:25:40.000000000 +0800
@@ -29,9 +29,9 @@
 " Load the Javascript indent script first, it defines GetJavascriptIndent().
 " Undo the rest.
 " Load base python indent.
-if !exists('*GetJavascriptIndent')
+" if !exists('*GetJavascriptIndent')
   runtime! indent/javascript.vim
-endif
+" endif
 let b:did_indent = 1
 
 setlocal indentexpr=HtmlIndent()
@@ -593,7 +593,7 @@
   "{{{
   if prevnonblank(v:lnum-1) == b:hi_indent.blocklnr
     " indent for first content line
-    return eval(b:hi_css1indent)
+    return shiftwidth() + indent(prevnonblank(v:lnum-1))
   endif
   return s:CSSIndent()
 endfunc "}}}
