--- /usr/share/nvim/runtime/indent/html.vim	2016-11-28 03:03:30.000000000 +0800
+++ _html.vim	2017-07-05 18:03:04.895170192 +0800
@@ -25,6 +25,10 @@
 if exists("b:did_indent") "{{{
   finish
 endif
+
+" Initial indent/javascript before set did_indent = 1
+ru! indent/javascript.vim
+
 let b:did_indent = 1
 
 setlocal indentexpr=HtmlIndent()
@@ -596,7 +600,7 @@
     return eval(b:hi_js1indent)
   endif
   if b:hi_indent.scripttype == "javascript"
-    return cindent(v:lnum)
+    return GetJavascriptIndent()
   else
     return -1
   endif
